templates/generic_show_form.html.twig

```diff
@@ -282,8 +282,9 @@
                      {% endif %}
 
                      {% if item.isField('date_expiration') %}
-                        {{ fields.datetimeField('date_expiration', item.fields['date_expiration'], __('Expiration date'), {
-                           'helper': __('Empty for infinite')
+                        {{ fields.dateField('date_expiration', item.fields['date_expiration'], __('Expiration date'), {
```

Comment:
`datetime` field type have to be used for `glpi_domains` and `glpi_computerantiviruses`. I guess this field has to be moved from generic form to specific forms.

Reply:
I am cancelling this change to deal with this case in another PR.